04-spatial-operations.Rmd

```diff
@@ -534,7 +545,7 @@ This is a good example of spatial aggregation: it is the geometry of the source
 This is illustrated using the base `aggregate()` function below:
 
 ```{r}
-nz_avheight = aggregate(nz_height, nz, FUN = mean)
+nz_avheight = aggregate(x = nz_height, nz, FUN = mean)
```

Comment:
As a stylistic preference, I've read that it's conventional for the first argument *not* to be stated and I think this is generally the case. However it does not harm and sometimes it's good to be explicit (thinking of the `, ,` debate) so leaving this as is. Thanks.

Reply:
Ah ok.The reason I did so here, was because in the text you explicitly mentioned x and y.

Reply:
That's a v. good reason!